lib/compat/wordpress-5.9/global-styles-assets.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
Should we name this file `script-loader.php` just to keep it in sync with WordPress Core and help the dev find the corresponding code if we ever want to make changes.

Reply:
That's better, yes!

Reply:
I was thinking that I'll port here also the code for hooking into the block editor settings (hence the file name), but I'd like that we mimic core's names. The other code should live in something like `block-editor.php` IIRC.

Reply:
https://github.com/WordPress/gutenberg/pull/37335/commits/998c827c099b0800516a4db5ceb5f7d027b4cd2c